bin/node/runtime/src/lib.rs

```diff
@@ -544,6 +550,8 @@ parameter_types! {
 		*RuntimeBlockLength::get()
 		.max
 		.get(DispatchClass::Normal);
+
+	pub const VoterSnapshotPerBlock: u32 = u32::max_value();
```

Comment:
dunno what this value is yet, but can we add a comment why it was selected as max value?

Also I think it should be `u32::MAX` as `max_value` is being deprecated.

Reply:
@kianenigma, thinking about it should we change this everywhere to 22,500 just in case we forget to update it in a follow up pr when we enable iterating the bags list?

Reply:
I think it is fine for now, but I noted in the companion that you should explain what this value is, and when and how it will be updated.

For substrate-node, since the bags list is already enabled, let's set it to 10k or sth. It really does not matter.